dfgviewer/Configuration/TypoScript/Plugins/kitodo.ts

```diff
@@ -97,14 +97,14 @@ plugin.tx_dlf_toc {
 		1.wrap = <ul class="toc">|</ul>
 		1.NO = 1
 		1.NO.stdWrap.crop = 55 | &nbsp;... | 1
-		1.NO.stdWrap.ifEmpty.field = type
+		1.NO.stdWrap.ifEmpty.field = tyoe
 		1.NO.stdWrap.ifEmpty.append = TEXT
 		1.NO.stdWrap.ifEmpty.append.fieldRequired = volume
 		1.NO.stdWrap.ifEmpty.append.field = volume
 		1.NO.stdWrap.ifEmpty.append.wrap = &nbsp;|
 		1.NO.stdWrap.dataWrap = | <span class="pagination">{field:pagination}</span>
 		1.NO.doNotLinkIt.field = doNotLinkIt
-		1.NO.ATagTitle.field = type
+		1.NO.ATagTitle.field = title
```

Comment:
In line 100 "type" is used as a fallback if "title" is empty. This fallback should be used in line 107, too. I.e.:

1.NO.ATagTitle.field = title // type